Themes/default/languages/Post.english.php

```diff
@@ -49,7 +49,7 @@
 $txt['attached'] = 'Attached';
 $txt['allowed_types'] = 'Allowed file types';
 $txt['attach_add'] = 'Add files';
-$txt['attach_drop_zone'] = 'Drag and drop your files here, or use the button to add files.';
+$txt['attach_drop_zone'] = 'Drag files here to attach them.';
```

Comment:
By clicking, I also get the prompt to attach my files.  This text should reflect drag or click here to attach.

Reply:
Has now been changed to "Click or drag files here to attach them."